src/applications/sign-in-changes/containers/InterstitialChanges.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { VaLink } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { selectProfile } from 'platform/user/selectors';
+import { AUTHN_SETTINGS } from '@department-of-veterans-affairs/platform-user/exports';
+import CreateAccount from '../components/CreateAccount';
+import AccountSwitch from '../components/AccountSwitch';
+
+export default function InterstitialChanges() {
+  const userEmail = useSelector(state => selectProfile(state)?.email);
```

Comment:
You could probably move `userEmail` to the `AccountSwitch` component since it's the only component using it.